spec/apibuilder-spec.json

```diff
@@ -270,9 +272,10 @@
 	{ "name": "organization", "type": "organization" },
 	{ "name": "application", "type": "application" },
 	{ "name": "version", "type": "string", "description": "The version of the service that we are importing", "example": "1.0.0" },
-	{ "name": "enums", "type": "[string]", "description": "Models made available by this import", "default": "[]" },
+	{ "name": "enums", "type": "[string]", "description": "Enums made available by this import", "default": "[]" },
 	{ "name": "unions", "type": "[string]", "description": "Unions made available by this import", "default": "[]" },
-	{ "name": "models", "type": "[string]", "description": "Models made available by this import", "default": "[]" }
+	{ "name": "models", "type": "[string]", "description": "Models made available by this import", "default": "[]" },
+    { "name": "annotations", "type": "[annotation]", "description": "Annotations made available by this import", "default": "[]" }
```

Comment:
type here should probably be `[string]`, since it's an import of an external spec.